src/sentry/static/sentry/app/views/settings/components/forms/radioField.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+import InputField from './inputField';
+import {growIn} from './styled/animations';
+
+class RadioField extends InputField {
+  static propTypes = {
+    id: PropTypes.number,
+    value: PropTypes.string,
+  };
+
+  onChange = (value, onChange, onBlur, e) => {
+    onChange(value, e);
+    onBlur(value, e);
+  };
+
+  isSelected = ({value, id}) => {
+    return value ? value === id : id === 0;
+  };
+
+  render() {
+    return (
+      <InputField
+        {...this.props}
+        field={({onChange, onBlur, value, disabled, ...props}) => (
+          <div>
+            {(props.choices() || []).map((choice, index) => {
```

Comment:
Looking at other components (select2field, multipleChoice<something>), choices is an array of tuples of `[value, label]` i.e.
```
choices: () => [
          [0, 'Always'],
          [1, 'Only On Deploys With My Commits'],
          [2, 'Never'],
];
```

We should try to keep structure of `choices` the same between these components.

Reply:
Right! I had it this way, then forgot why, then changed it this lol. Will undo.